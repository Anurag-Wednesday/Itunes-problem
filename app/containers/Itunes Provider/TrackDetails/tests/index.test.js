/**
 *
 * Tests for TrackDetails
 *
 *
 */

import React from 'react';
import { renderProvider } from '@utils/testUtils';
// import { fireEvent } from '@testing-library/dom'
import { TrackDetailsTest as TrackDetails } from '../index';

describe('<TrackDetails /> container tests', () => {
  // let submitSpy

  beforeEach(() => {
    // submitSpy = jest.fn()
  });
  it('should render and match the snapshot', () => {
    const { baseElement } = renderProvider(<TrackDetails />);
    expect(baseElement).toMatchSnapshot();
  });
});
