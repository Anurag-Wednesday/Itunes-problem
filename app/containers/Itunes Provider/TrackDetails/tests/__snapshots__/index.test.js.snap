// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrackDetails /> container tests should render and match the snapshot 1`] = `
<body>
  <div>
    <div>
      TrackDetails
    </div>
  </div>
</body>
`;
